supporting_content/use_cases/compare_billed_cost_per_subaccount_to_budget.md

```diff
@@ -0,0 +1,54 @@
+# Use Case
+
+Compare billed cost per subaccount to budget
+
+## Use Case ID
+14383
```

Comment:
Is this our number or the UC library's number? Should we reset numbers for the spec? I'm not sure how we jumped up to 14.3k use cases already 😜